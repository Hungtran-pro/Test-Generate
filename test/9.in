33
1783
1550
1888
1812
1799
1859
1889
1327
1424
1758
1105
1244
1538
1139
1653
1217
1666
1248
1889
1696
1303
1322
1401
1109
1505
1314
1312
1162
1486
1997
1724
1905
1217
