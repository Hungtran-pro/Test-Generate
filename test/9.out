144794747
775999682
788822458
351622543
156317596
857628789
666760983
25403431
24085959
906727303
484688741
478970324
59445945
216615959
873104418
806771845
268277194
160115778
666760983
113639206
784128804
589551752
74258288
721163499
526927074
148950919
120979802
397440311
455573080
809903487
29751567
404711531
806771845
